### ISSUE TITLE
support `Symbol` type event name?

### ISSUE THREAD

**Issue by mrdulin** [mroderick/PubSubJS#117]
```js
class PubsubEvents {
  static OPEN_POPUP = Symbol('OPEN_POPUP');
}
```

purpose: Just want to avoid event name conflict.

**Comment by AnthonyYY** [comment]
I think you should pull a request

**Comment by mroderick** [comment]
A pull request would be very welcome. 

Please note that PubSubJS is meant to be compatible with ES3, so you should protect the code accordingly. It's not that difficult to do feature detection in the tests, to see if the runtime supports `Symbol` or not.

**Comment by mroderick** [comment]
This was implemented in #141 and published to npm as `pubsub-js@1.7.0`
